config/200-config.yaml

```diff
@@ -44,3 +44,8 @@ data:
     # in the context of services should be logged. Readiness
     # probes etc. must be configured via the bootstrap config.
     enable-service-access-logging: "true"
+
+    # Specifies whether to use proxy-protocol in ordrer to to safely
+    # transport connection information such as a client's address
+    # across multiple layers of TCP proxies.
+    proxy-protocol: "false"
```

Comment:
Do we want to use this one global configuration for the proxy-protcol?
Is it not better to introduce an annotation like `networking.knative.dev/proxy-protocol: true` to Ingress object? Then, I wonder we can add a few tests to verify the proxy-protocol instead of setup another cluster :thinking:

Reply:
If I understood correctly, we wanna have an L4 proxy that suppports proxy protocol, but headers will be forwarded only if the ksvc/ingress is asking for them ...

Unless I'm missing something, to enable proxy protocol with envoy, we should configure as a listener filter (ListenerFilter at the listener level) section and it cannot be configured by ksvc/ingress